laser_filter_node:
  ros__parameters:
    scan_filter_chain: [
      {
        name: 'narrow_fov_filter',
        type: 'laser_filters/LaserScanAngularBoundsFilter',
        params: {
          angle_min: -30,
          angle_max: 30
        }
      }
    ]
